tests/test_indieweb_utils.py

```diff
@@ -0,0 +1,43 @@
+# Partial test coverage is implemented for various functions in the library
+# Assistance is requested to achieve full test coverage
+import pytest
+
+from indieweb_utils import *
+
+
+@pytest.mark.parametrize(
```

Comment:
Use pytest parametrize so each combination is treated as its own test, rather the one big test.

Reply:
Sounds good, thanks!